worlds/peaks_of_yore/__init__.py

```diff
@@ -0,0 +1,233 @@
+    tutorials = [Tutorial(
+        "Multiworld Setup Guide",
+        "A guide to setting up the Peaks of Yore Mod.",
+        "English",
+        "setup_en.md",
+        "setup/en",
+        ["c0der23"]
+    )]
+    option_groups = poy_option_groups
+
+
+class PeaksOfWorld(World):
+    """
+    Peaks of Yore is a first-person physics-based "climb-em-up" adventure set in 1887.
+    Steel your nerves and perfect your climbing skills as you ascend the rock wall, traverse difficult routes,
+    and encounter many challenges and obstacles.
+    """
+    game = "Peaks of Yore"
+    options_dataclass = PeaksOfYoreOptions
+    options: PeaksOfYoreOptions
+    web = PeaksOfWeb()
+    item_name_to_id = {item.name: item.id for item in full_item_list}
+    location_name_to_id = {location.name: location.id for location in full_location_list}
+    topology_present = True
+    location_count: int
+    peaks_in_pool: list[str]
+    artefacts_in_pool: list[str]
+
+    def __init__(self, multiworld: MultiWorld, player: int):
+        super().__init__(multiworld, player)
+        self.location_count = 0
+        self.peaks_in_pool = []
+        self.artefacts_in_pool = []
+
+    def create_item(self, name: str) -> Item:
+        item_entry = [item for item in full_item_list if item.name == name][0]
+        return PeaksOfYoreItem(name, item_entry.classification, self.item_name_to_id[name], self.player)
+
+    def get_filler_item_name(self) -> str:
+        choices = ["Extra Rope", "Extra Coffee", "Extra Chalk", "Extra Seed"]
+        return self.random.choice(choices)
+
+    def generate_early(self) -> None:
+        if self.options.start_with_barometer:
+            self.multiworld.push_precollected(self.create_item("Barometer"))
+
+        if self.options.start_with_oil_lamp:
+            self.multiworld.push_precollected(self.create_item("Oil Lamp"))
+
+        starting_book_options: dict[str, Toggle] = {
+            "Fundamentals Book": self.options.enable_fundamental,
+            "Intermediate Book": self.options.enable_intermediate,
+            "Advanced Book": self.options.enable_advanced,
+            "Expert Book": self.options.enable_expert,
+            "Random": None
+        }
+
+        book_names: list[str] = [s for s, t in starting_book_options.items()]
+        enabled_books: list[str] = [b for b, v in starting_book_options.items() if b != "Random" and v.value]
+        start_book: str = ""
+
+        if len(enabled_books) == 0:
+            raise Exception("Player " + self.player_name + " has not selected any books!")
+
+        if self.options.starting_book.value == StartingBook.option_random_book:
+            start_book = self.random.choice(enabled_books)
+        else:
+            start_book = book_names[self.options.starting_book.value]
+
+        if start_book not in enabled_books:
+            print("book " + start_book + "not enabled")
+            start_book = enabled_books[0]
+            print("selecting " + start_book)
```

Comment:
Please either remove these print statements, or use the logging module (logging.info, logging.warning, logging.debug, logging.error)

Reply:
(If you use logging, then only statements higher than the specified --log_level will show up, which you can pass as a CLI arg)